packages/runtime-common/indexer.ts

```diff
@@ -499,6 +504,144 @@ export class Indexer {
     ];
   }
 
+  private buildPrerenderedInstancesQuery(everyCondition: CardExpression) {
```

Comment:
it looks like this query is running against the WIP index too. we'll need to scope this to just the production index such that you are only querying records whose realm version is the maximum for the url that is less than or equal to the current version for the realm. we have a help for this what you should be using.

Reply:
also, you should explicitly add a unit test that asserts that you are not searching against items in the WIP index